sources/api/netdog/src/net_config/test_macros/bonding.rs

```diff
@@ -75,6 +75,20 @@ macro_rules! bonding_tests {
                 let rendered = render_config_template(bad);
                 assert!(deserialize_config(&rendered).is_err())
             }
+
+            #[test]
+            fn mac_as_identifier() {
+                let bad = net_config().join("mac_as_identifier.toml");
```

Comment:
It occurs to me that we don't do a good job separating bad from good configs in that directory by name. One must look at the test name to know if it should or shouldn't be valid. If someone just looked through and saw these files, they might mistake them for valid. There really isn't any explanation why they are bad.

Reply:
I suppose that's true.  Probably something we could take care of separately if we're concerned about it.